tests/s3_bucket/test_github_content_collections.py

```diff
@@ -65,48 +65,52 @@ def test_github_content_collections(git_content_repos, headers_data, abl_books_u
                     resp_content = collections_resp.text
 
                     soup = BeautifulSoup(resp_content, "xml")
+                    col_collection = soup.find_all("collection")
 
-                    md_title = soup.find_all("md:title")
-                    md_license = soup.find_all("md:license")
-                    content = soup.find_all("col:content")
+                    for col in col_collection:
 
-                    try:
+                        try:
+                            assert col.find("md:title").text is not None
```

Comment:
This might be a problem here and on line 86: col.find(...).text will return an empty string, not None. 

If you want to check that the element has content, it is better to test the length (I usually throw in a `.strip()` too in case someone has included a space inside an element.
```python
>>> col = BeautifulSoup('<root><a></a></root>')
>>> col.find('a').text
''
>>> # this one would never raise an assertion error
>>> assert col.find('a').text is not None
>>>
>>> assert len(col.find('a').text.strip()) > 0
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError
>>>
>>> col = BeautifulSoup('<root><a>        </a></root>')
>>> assert len(col.find('a').text.strip()) > 0
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError
```

Reply:
ok. first I tried len than I felt that None is better... :)